src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLibrary.java

```diff
@@ -55,14 +60,54 @@
      */
     private final Random random = new Random();
 
+    /**
+     * Study Item list, loaded from config, Typle<int,int> First is the study chance increase,second is the breakchance
+     */
+    private final List<StudyItem> studyItems;
+
     /**
      * Instantiates the building.
+     *
      * @param c the colony.
      * @param l the location.
      */
     public BuildingLibrary(final Colony c, final BlockPos l)
     {
         super(c, l);
+
+        studyItems = parseFromConfig();
```

Comment:
we should read and write these from and to NBT as well, else we lose the break progress on world unload/load